### ISSUE TITLE
Average CPU temperature shows 0 but individual temps have different values

### ISSUE THREAD

**Issue by mozbius** [mpurses/Sonder#12]
I guess there's a bug that is miscalculating the average temp for the CPU?

Average:
![average](https://user-images.githubusercontent.com/54375590/112693299-ca761500-8e56-11eb-9194-d8981c402593.png)

Individuals:
![individuals](https://user-images.githubusercontent.com/54375590/112693306-cc3fd880-8e56-11eb-81b4-ae798ebeba58.png)

Sonder setup:
![HWInfoAndCore](https://user-images.githubusercontent.com/54375590/112693311-cea23280-8e56-11eb-9e8f-0a5c7419e4c9.png)

**Comment by mpurses** [comment]
With Hwinfo and coretemp both running, Hwinfo will take priority and coretemp will be ignored. 

In the "Edit HWiNFO CPU ID Settings" window, make sure the proper package temp is linked (along with all the others too at this time). Every system is different and Hwinfo will probably try to hide it in a different place on your computer, but below is where it is shown on my laptop:

![Screenshot 2021-04-02 081439](https://user-images.githubusercontent.com/642218/113418969-1e6b8700-938c-11eb-9ad5-b60639fbb2fd.png)

Usually the best way to tell if it is the proper sensor is looking at the values for that sensor.

**Comment by mozbius** [comment]
Thank you for the help. Much appreciated. After some time playing on the Hackintosh I went back to Windows 10 to give a second look at Sonder. I have looked around and I don't appear to have a "CPU Package" anywhere in there but "CPU Package Temperature (degrees)" is showing on the right. So what I did is select a random Core # on the left (which shows the "Type" field to be a temperature and then clicked on "Update" at the bottom right and now the Average CPU temperature gets updated with the average. Thanks again in helping fix up Sonder on my PC. Great tweak!!!

NOTE (if it can be of any help for others): I have uninstalled CoreTemp, updated Rainmeter to 4.4.0.3447 and running HWiNFO64 6.24-4120.
